### PR TITLE
test aws arm images via cloud API

### PR DESCRIPTION
**Pull request by diaasami**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: test/cases/api/common/aws.sh [hunk 1]
```diff
@@ -17,11 +17,15 @@ function installClient() {
   $AWS_CMD --version
 
   if ! hash govc; then
```
Comment (ondrejbudai) [inline test/cases/api/common/aws.sh:19]: My god, I forgot we are reusing the same code for testing aws and vsphere. I wonder if we can split that so we don't need to install `govc` every time we test aws images. :thinking: Just food for thought.

Comment (diaasami) [reply test/cases/api/common/aws.sh:19]: Makes sense, I was wondering what `govc` is doing in `aws.sh`

Comment (diaasami) [reply test/cases/api/common/aws.sh:19]: I am fixing this in a separate PR, but would like to get an initial review.

**Review by ondrejbudai — APPROVED** [review]

**Comment by ondrejbudai** [comment]
This is perfect! :)

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -582,6 +582,10 @@ API:
         RUNNER:
           - aws/fedora-35-x86_64
           - aws/fedora-36-x86_64
+      - IMAGE_TYPE: ["aws"]
+        RUNNER:
+          - aws/rhel-8.6-ga-aarch64
```
Comment (ondrejbudai) [inline .gitlab-ci.yml:587]: Oh, just one thing, can we enable this also on rhel-9.0? I would like to have coverage on both distros. Can be done in a follow-up.

**Review by jrusz — APPROVED** [review]
LGTM, so are we merging this and adding 9.0-ga in a followup?

**Comment by diaasami** [comment]
> LGTM, so are we merging this and adding 9.0-ga in a followup?

@jrusz Yes let's do that

File: .gitlab-ci.yml (hunk 2 above)
Comment (jrusz) [reply .gitlab-ci.yml:587]: @diaasami agreed to add rhel-9.0 in a followup PR.

**Pull request merged**